packages/bpk-component-button/index.js

```diff
@@ -17,5 +17,15 @@
  */
 
 import BpkButton from './src/BpkButton';
+import {
+  primaryThemeAttributes,
+  secondaryThemeAttributes,
+  linkThemeAttributes,
```

Comment:
I can't see this referred to anywhere else - is it meant to be here?

Reply:
Ah nice catch! I removed the link theme stuff so that is an orphan.